### DISCUSSION TITLE
ERP with BudiBase it is possible ?

### DISCUSSION THREAD

**Discussion by massacarlos** [Budibase/budibase#6559] [General]
Can I create an ERP (150/200 tables) accessing an existing database in SQLServer?

**Comment by melohagan** [comment]
Hi @massacarlos 

You can certainly connect 150-200 tables via the SQL Server connector, although Budibase isn't advertised as an ERP first solution e.g. SAP, Oracle, and Microsoft Dynamics.

You can build a variety of business applications and internal tools, but it may be worth contacting sales (https://budibase.com/contact/) if you want to lay out your requirements in more detail.

**Reply by massacarlos** [reply]
I have an ERP (children's soccer school) made with Bubble and as a starting point for learning BudiBase I would be developing the application again from scratch. I also have another ERP (import/export) made with Genexus that should be migrated to the web. Is it possible to face these challenges with BudiBase? Thanks for listening.

**Reply by melohagan** [reply]
Yes, should be child's play!
